test/java/seedu/addressbook/util/TestUtil.java

```diff
@@ -60,4 +65,13 @@ public static Person generateTestPerson() {
             return null;
         }
     }
+
+    /**
+     * Asserts whether the text in the two given files are the same.
+     */
+    public static void assertTextFilesEqual(Path path1, Path path2) throws IOException {
+        List<String> list1 = Files.readAllLines(path1, Charset.defaultCharset());
+        List<String> list2 = Files.readAllLines(path2, Charset.defaultCharset());
+        assertEquals(list1, list2);
```

Comment:
Maybe make this
```java
assertEquals(String.join("\n", list1), String.join("\n", list2));
```
so that the assertion error message is more meaningful.

(If we are comparing lists, JUnit prints the full contents of the lists. If we are comparing strings, JUnit only prints the differences).